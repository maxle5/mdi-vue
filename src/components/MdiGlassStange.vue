
<template>
  <svg :width="size" :height="size" viewBox="0 0 24 24">
      <path d="M8,2H16V22H8V2M10,4V7H14V4H10Z" style="fill: currentColor" />
  </svg>
</template>

<script lang="ts">
  import { defineComponent } from 'vue';

  export default defineComponent({
      name: "MdiGlassStange",
      props: {
          size: { 
              type: Number, 
              default: 24 
          }
      },
  })
</script>
