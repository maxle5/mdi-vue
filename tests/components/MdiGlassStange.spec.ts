
import { mount } from "@vue/test-utils";
import { expect, test } from "vitest";
import MdiGlassStange from "../../src/components/MdiGlassStange.vue";

test("MdiGlassStange snapshot", () => {
  const wrapper = mount(MdiGlassStange, {});
  expect(wrapper.html()).toMatchSnapshot();
});
