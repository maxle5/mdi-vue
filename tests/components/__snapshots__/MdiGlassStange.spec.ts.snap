// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`MdiGlassStange snapshot 1`] = `
"<svg width="24" height="24" viewBox="0 0 24 24">
  <path d="M8,2H16V22H8V2M10,4V7H14V4H10Z" style="fill: currentColor;"></path>
</svg>"
`;
